WeaponPack/src/main/java/com/github/sef24sp4/weaponpack/AbstractBulletControlSystem.java

```diff
@@ -5,7 +5,7 @@
 import com.github.sef24sp4.common.projectile.ProjectileSPI;
 import com.github.sef24sp4.common.services.IEntityProcessingService;
 
-public abstract class AbstractBulletControlSystem implements IEntityProcessingService, ProjectileSPI {
```

Comment:
ProjectileSPI is removed, and its related Import.

Might not be seen here, but please refer to the next commit message.